$NetBSD: patch-ab,v 1.3 2015/12/29 23:34:43 dholland Exp $

XXX: it is unclear what the point of this patch is.

--- Demos/Makefile.orig
+++ Demos/Makefile
@@ -34,13 +34,15 @@
 	do \
 	  chmod 0755 $(DEMODIR)/`basename $$i`; \
 	done
-	mkdir $(DEMODIR)/Html-Demos; chmod 0755 $(DEMODIR)/Html-Demos 
+	-mkdir $(DEMODIR)/Html-Demos
+	-chmod 0755 $(DEMODIR)/Html-Demos 
 	cp Html-Demos/*.html $(DEMODIR)/Html-Demos
 	chmod 0644  $(DEMODIR)/Html-Demos/*.html
-	mkdir $(DEMODIR)/Html-Demos/Images; chmod 0755 $(DEMODIR)/Html-Demos/Images
+	-mkdir $(DEMODIR)/Html-Demos/Images
+	-chmod 0755 $(DEMODIR)/Html-Demos/Images
 	cp Html-Demos/Images/*.gif  $(DEMODIR)/Html-Demos/Images
 	chmod 0644  $(DEMODIR)/Html-Demos/Images/*
-	(cd $(DEMODIR)/Html-Demos; ln -s ../amib.stklos ../stklos-widgets .)
+	(cd $(DEMODIR)/Html-Demos; ln -fs ../amib.stklos ../stklos-widgets .)
 	-if [ ! -d $(DEMODIR)lib ] ; then mkdir -p $(DEMODIR)/lib; fi
 	$(CP) ../Contrib/STk-wtour/lib/*.xbm $(DEMODIR)/lib
 	chmod 0644 $(DEMODIR)/lib/*
